account_asset_management/models/account_move.py

```diff
@@ -30,13 +30,18 @@ class AccountMove(models.Model):
     asset_count = fields.Integer(compute="_compute_asset_count")
 
     def _compute_asset_count(self):
-        for rec in self:
-            assets = (
-                self.env["account.asset.line"]
-                .search([("move_id", "=", self.id)])
-                .mapped("asset_id")
+        if len(self) < 100:
+            for move in self:
+                move.asset_count = len(move.line_ids.filtered("asset_id"))
+        else:
+            self.asset_count = 0
```

Comment:
Why not doing this always?

Reply:
This is unnecessary in the other condition